# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1245016360 -3600
#      Sun Jun 14 22:52:40 2009 +0100
# Node ID 44377339a398888f343ef9a0a2be8e558228f529
# Parent  3da821b161e93df42b356585b1d23ea05111db38
Move changelog entry to the correct position.

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,27 @@
+2009-06-14  Michael Goffioul  <michael.goffioul@gmail.com>
+
+	* graphics.h.in (axes::properties::xtick, axes::properties::ytick,
+	axes::properties::ztick, axes::properties::xticklabelmode,
+	axes::properties::yticklabelmode, axes::properties::zticklabelmode):
+	Add 'u' modifier to enable updaters.
+	(axes::properties::calc_ticklabels): New function to update tick
+	labels.
+	(axes::properties::update_xtick, axes::properties::update_ytick,
+	axes::properties::update_ztick,
+	axes::properties::update_xticklabelmode,
+	axes::properties::update_yticklabelmode,
+	axes::properties::update_zticklabelmode): Add updaters to update tick
+	labels correctly.
+	(axes::properties::update_xlim, axes::properties::update_ylim,
+	axes::properties::update_zlim): Update tick labels.
+	* graphics.cc (axes::properties::calc_ticklabels): New function to
+	update tick labels.
+
 2009-06-14  Jaroslav Hajek  <highegg@gmail.com>
 
 	* DLD-FUNCTIONS/lookup.cc (Flookup): Support character array lookup.
 
 2009-06-12  John W. Eaton  <jwe@octave.org>
 
 	* ov-fcn-handle.cc (make_fcn_handle): Accept operators which have
 	corresponding functions and make function handles using the names.
@@ -186,35 +205,16 @@ 2009-05-25  Jaroslav Hajek  <highegg@gma
 	(Fquit): Raise error if quitting is not allowed.
 	* octave.cc (octave_main): if running as embedded, disable quit by
 	default.
 
 2009-05-25  Jaroslav Hajek  <highegg@gmail.com>
 
 	* variables.cc (do_who): Only output symbols with a defined value.
 
-2009-06-14  Michael Goffioul  <michael.goffioul@gmail.com>
-
-	* graphics.h.in (axes::properties::xtick, axes::properties::ytick,
-	axes::properties::ztick, axes::properties::xticklabelmode,
-	axes::properties::yticklabelmode, axes::properties::zticklabelmode):
-	Add 'u' modifier to enable updaters.
-	(axes::properties::calc_ticklabels): New function to update tick
-	labels.
-	(axes::properties::update_xtick, axes::properties::update_ytick,
-	axes::properties::update_ztick,
-	axes::properties::update_xticklabelmode,
-	axes::properties::update_yticklabelmode,
-	axes::properties::update_zticklabelmode): Add updaters to update tick
-	labels correctly.
-	(axes::properties::update_xlim, axes::properties::update_ylim,
-	axes::properties::update_zlim): Update tick labels.
-	* graphics.cc (axes::properties::calc_ticklabels): New function to
-	update tick labels.
-
 2009-05-22  Michael Goffioul  <michael.goffioul@gmail.com>
 
 	* toplev.h (main_loop): Tag with OCTINTERP_API.
 	* input.h (octave_read, get_input_from_file, get_input_from_stdin):
 	Ditto.
 	* lex.h (create_buffer, current_buffer, switch_to_buffer,
 	delete_buffer, restore_input_buffer, delete_input_buffer): Ditto.
 
